pages/login.tsx

```diff
@@ -0,0 +1,122 @@
+import Image from "next/image";
+import React, { ChangeEvent, useState } from "react";
+import { login as translate } from "../public/locales/fa.json";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const Login = () => {
```

Comment:
- Check out naming conventions in `docs` repo.
- It would be better if you specify the type of your page component by using `NextPage`.
```suggestion
const LoginPage: NextPage = () => {
```